api/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,5 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.6
```

Comment:
all these eclipse files need to be removed, you needed to add a .gitignore file to ignore them before commiting